msal/tests/dev apps/UWP/MainPage.xaml.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Microsoft.Identity.Client;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
+namespace UWP
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        private IPublicClientApplication _pca;
+        private readonly static string ClientID = "0615b6ca-88d4-4884-8729-b178178f7c27";
+        private readonly static string Authority = "https://login.microsoftonline.com/organizations"; // common will not work for WIA and U/P but it is a good test case
+        private readonly static IEnumerable<string> Scopes = new[] { "user.read" };
+
+        public MainPage()
+        {
+            this.InitializeComponent();
+
+            _pca = new PublicClientApplication(ClientID, Authority);
+        }
+
+
+        private async void AcquireTokenIWA_ClickAsync(object sender, RoutedEventArgs e)
+        {
+            AuthenticationResult result = null;
+            try
+            {
+                //result = await _pca.AcquireTokenByIntegratedWindowsAuthAsync(Scopes);
+                 result = await _pca.AcquireTokenByIntegratedWindowsAuthAsync(Scopes, "bogavril@microsoft.com"); // can also use this overload
```

Comment:
>"bogavril@microsoft.com" [](start = 86, length = 24)

move to field on the form and make a selector on the form to try either with or with-out the username (upn)